clap_builder/Cargo.toml

```diff
@@ -67,6 +67,8 @@ anstyle = "1.0.8"
 terminal_size = { version = "0.4.0", optional = true }
 backtrace = { version = "0.3.73", optional = true }
 unicode-width = { version = "0.2.0", optional = true }
```

Comment:
New depts. serde_yml is for deserializing the external config file where the hardcoded texts are extracted to.